src/Kunstmaan/ArticleBundle/Router/TagCategoryRouter.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Kunstmaan\ArticleBundle\Router;
+
+use Kunstmaan\NodeBundle\Router\SlugRouter;
+use Symfony\Component\Routing\Route;
+use Symfony\Component\Routing\RouteCollection;
+use Symfony\Component\Translation\TranslatorInterface;
+use Symfony\Component\VarDumper\VarDumper;
+
+class TagCategoryRouter extends SlugRouter
+{
+    /**
+     * @return \Symfony\Component\Routing\RouteCollection
+     */
+    public function getRouteCollection()
+    {
+        if (! is_null($this->routeCollection)) {
+            return $this->routeCollection;
+        }
+        $this->routeCollection = new RouteCollection();
+
+        $extendParameters = array('category' => null, 'tag' => null);
```

Comment:
Stylistic: phpstorm is nagging me to use the short notation [] as the new way of making arrays